tox.ini

```diff
@@ -42,7 +43,7 @@ skip_install = true
 deps =
     flake8
     flake8-bugbear
-commands = flake8 src/ tests/
+commands = flake8 --extend-ignore E501 src/ tests/
```

Comment:
E501 is already ignored above

Reply:
ISTG E501 wasn't getting ignored the last time I tried it out using tox.

Removing `--extend-ignore E501`.